net/udpbroadcastrelay/Makefile

```diff
@@ -0,0 +1,5 @@
+udpbroadcastrelay: main.c
+	gcc -g main.c -o udpbroadcastrelay
```

Comment:
should be the variable CC which can be gcc or clang in most cases - in linux this is GCC, on BSD it is LLVM (clang).

Reply:
Is done.